ShareX.UploadersLib/Helpers/ResponseHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿using ShareX.HelpersLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ShareX.UploadersLib
+{
+    public class ResponseHelpers : Uploader
+    {
+        public static string UrlRedirectLink(string url)
+        {
+            if (!URLHelpers.IsValidURL(url)) return null;
+
+            ResponseHelpers responseHelper = new ResponseHelpers();
+            var response = responseHelper.GetResponse(HttpMethod.GET, url);
+
+            if (response == null) return null;
+
+            if (response.ResponseUri.ToString().ToLower() == url.ToLower()) return null;
+
+            return response.ResponseUri.ToString();
```

Comment:
I see there's a mix of tabs and spaces here, could you normalize them into spaces? Thanks

Reply:
Commit "Convert all tabs to spaces" should have resolved this, please let me know if you are still seeing tabs